### ISSUE TITLE
Employer request Job Seeker application

### ISSUE THREAD

**Issue by joaopapereira** [AgileVentures/MetPlus_tracker#49]
As an Employer
When I search for Job Seekers
And find a good match
I request to the Job Developer to contact that Job Seeker

**Comment by patmbolger** [comment]
see JD:E/5

**Comment by patmbolger** [comment]
This is gated by #50.

**Comment by joaopapereira** [comment]
This story is very old, so we created a new story
Related with #411
